README.md

```diff
@@ -178,6 +178,28 @@ as the `:archive_mode` property of the `options` parameter passed into the
 For more information on archiving, see the
 [OpenTok archiving](https://tokbox.com/opentok/tutorials/archiving/) programming guide.
 
+## Signaling
+
+You can send a signal using the `opentok.signal.send(session_id, connection_id, opts)` method.  
+If `connection_id` is nil or an empty string, then the signal is send to all valid connections in the 
+session. 
+
+The hash `opts` field is optional. An example of it is as follows:
```

Comment:
The opts field is optional?

Reply:
Theoretically yes , but it is a rare case.  Hence I am rewording it as "An example of `opts` field can be as follows:"